tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
 pytest
 pytest-cov
-conan==1.62.0
-velocitas-lib==0.0.4
+conan>2
```

Comment:
Huh, your're future oriented ^^ But maybe we should stick to version2, for now:
```suggestion
conan==2.*
```

Reply:
yeah, didn't think that conan 3 will be released soon :D